packages/vat-data/src/types.d.ts

```diff
@@ -23,22 +23,41 @@ type ActualBehavior<B> = {
   [Facet in keyof B]: FunctionsMinusContext<B[Facet]>;
 };
 
-type KindContext<S, B> = { state: S; facets: ActualBehavior<B> };
-
 interface KindDefiner {
+  <P, S, F>(
+    tag: string,
```

Comment:
the changes in this file make sense to me in a vague sort of way, but I don't know the defineKind API well enough to approve it. paging @warner ? @FUDCo ?

Reply:
Pulled out to https://github.com/Agoric/agoric-sdk/pull/5150